.storybook/preview-head.html

```diff
@@ -186,3 +186,22 @@
     box-sizing: border-box;
   }
 </style>
+
+<style>
+  button {
+    cursor: pointer;
+    color: inherit;
+    background-color: transparent;
+    transition: all 150ms ease-in-out;
```

Comment:
na pewno chcemy transition all na buttonie?

Reply:
Why not?
1. Łatwo nadpisać
2. To standardowe transition jakiego używamy chyba

Być może nie chcemy `all`, to inna kwestia.